README.md

```diff
@@ -22,9 +22,11 @@ The connector is great for getting started with Conduit but also for certain typ
 
 ### Configuration
 
-| name          | description                                                                                | required | Default |
-|---------------|--------------------------------------------------------------------------------------------|----------|---------|
-| recordCount   | Number of records to be generated. -1 for no limit.                                        | false    | "-1"    |
-| readTime      | The time it takes to 'read' a record.                                                      | false    | "0s"    |
-| fields        | A comma-separated list of name:type tokens, where type can be: int, string, time, bool.    | true     | ""      |
-| format        | Format of the generated payload data: raw, structured.                                     | false    | "raw"   |
+| name         | description                                                                                | required | Default             |
+|--------------|--------------------------------------------------------------------------------------------|----------|---------------------|
+| recordCount  | Number of records to be generated. -1 for no limit.                                        | false    | "-1"                |
+| readTime     | The time it takes to 'read' a record.                                                      | false    | "0s"                |
+| sleepTime    | The time the generator 'sleeps' before it starts generating records. Must be non-negative. | false    | "0s"                |
```

Comment:
The description sounds to me like the field controls how long the generator waits before starting to generate records _the first time_, while in reality the sleep time will be applied repeatedly after `generateTime` is reached. I think we should clarify this.